ssg/build_cpe.py

```diff
@@ -153,6 +153,12 @@ def __init__(self, cpeitem_data):
         self.name = cpeitem_data["name"]
         self.title = cpeitem_data["title"]
         self.check_id = cpeitem_data["check_id"]
+        self.bash_conditional = ""
+        if "bash_conditional" in cpeitem_data.keys():
+            self.bash_conditional = cpeitem_data["bash_conditional"]
+        self.ansible_conditional = ""
+        if "ansible_conditional" in cpeitem_data.keys():
+            self.ansible_conditional = cpeitem_data["ansible_conditional"]
```

Comment:
If  you look for a key in a dictionary, you don't have to use `x in d.keys()` it's just `x in d`. Also, you can use `.get()` to which gives a default value if a key isn't present in the dictionary.

```suggestion
        self.bash_conditional = cpeitem_data.get("bash_conditional", "")
        self.ansible_conditional = cpeitem_data.get("ansible_conditional", "")
```